modules/tensor_mechanics/src/materials/abaqus/AbaqusUMATStress.C

```diff
@@ -27,6 +27,11 @@ AbaqusUMATStress::validParams()
   params.addClassDescription("Coupling material to use Abaqus UMAT models in MOOSE");
   params.addRequiredParam<FileName>(
       "plugin", "The path to the compiled dynamic library for the plugin you want to use");
+  params.addRequiredParam<bool>(
+      "use_one_based_indexing",
+      "MOOSE is transitioning to supporting 1-based indexing to ensure maximum "
+      "compatibility with UMAT plugins. During the transition period we require "
+      "users to specify this parameter to acknowledge the change.");
```

Comment:
Since it looks like that "one_based" is the way to go here, one option is to not require it and default to the _right_ way. Also "MOOSE is transitioning" -> "MOOSE has transitioned": Is there anything that remains?

Reply:
The statement about MOOSE transitioning to one-based numbering isn't really true, is it?  That makes it sound like we're switching all of MOOSE's internal node numbering to start at 1 across the board, which I doubt is the case. Also, this parameter description doesn't actually say what setting the parameter to true or false actually does. Suggested re-wording:

Parameter to control whether indexing for element and integration points as presented to UMAT models is based on 1 (true) or 0 (false).  This does not affect internal MOOSE numbering. The option to use 0-based numbering is deprecated and will be removed soon.

Reply:
Made the change